src/bellpepper/test_shape_cs.rs

```diff
@@ -14,6 +14,7 @@ use ff::{Field, PrimeField};
 #[derive(Clone, Copy)]
 struct OrderedVariable(Variable);
 
+#[allow(dead_code)]
```

Comment:
I think the clippy error for this might be because of the latest Rust toolchain. So, we should fix.

However, I'm not sure allowing dead code is the right way to fix this problem (allowing dead_code is just ignoring the issue rather than fixing it!)